components/profile_popover/profile_popover.tsx

```diff
@@ -604,12 +605,14 @@ ProfilePopoverState
                     className='popover__row first'
                 >
                     <button
-                        className='style--link'
+                        id='editProfileButton'
+                        type='button'
+                        className='btn'
                         onClick={this.handleEditAccountSettings}
                     >
-                        <LocalizedIcon
-                            className='fa fa-pencil-square-o'
-                            title={{
+                        <AccountOutlineIcon
+                            size={16}
+                            aria-label={{
```

Comment:
Need to use the localization function here, this is just an object

Reply:
yep you are right 😅